src/site/_includes/components/Tooltip.js

```diff
@@ -18,20 +18,9 @@ const {html} = require('common-tags');
 const md = require('markdown-it')();
 
 module.exports = (content, alignment) => {
-  switch (alignment) {
-    case 'left':
-      alignment = 'w-tooltip--left';
-      break;
-    case 'right':
-      alignment = 'w-tooltip--right';
-      break;
-    default:
-      alignment = '';
-  }
-
   return html`
-    <span role="tooltip" class="w-tooltip ${alignment}">
-      ${md.renderInline(content)}
+    <span class="tooltip" data-alignment="${alignment}">
```

Comment:
Let's remove this component instead. I don't see the need for an author to use this in Markdown documents. Searching for `{% Tooltip` inside the codebase seems to show it also isn't used currently.